### ISSUE TITLE
informix dialect is missing.

### ISSUE THREAD

**Issue by CodingKrishna** [confluentinc/kafka-connect-jdbc#576]
Failed to run query for table TimestampIncrementingTableQuerier{table="db_xyz "." informix "." test", query='null', topicPrefix='abc-', incrementingColumn='testid', timestampColumns=[]}: {} (io.confluent.connect.jdbc.source.JdbcSourceTask)
java.sql.SQLSyntaxErrorException: A syntax error has occurred.

**Comment by rhauch** [comment]
Correct, there is no Informix dialect yet. If the general JDBC dialect does not work, then we'd need to add an Informix dialect that has the Informix-specific behaviors.

**Comment by dungba82** [comment]
Is somebody have a solution for this issues? Please help.

**Comment by vigneshtvp** [comment]
Hi @dboyce , @rhauch,@stheppi,
I have a fix for this please do let me know whether i need to contributor to add this feature.

**Comment by maximilianobl** [comment]
@vigneshtvp  could you share your jdbc with the modifications in some repository? I would greatly appreciate it. :)
